docs/using-radicle/pushing-and-pulling-changes.md

```diff
@@ -23,44 +23,44 @@ It can be pushed to Radicle with:
+[insert picture of Radicle ID in seednode UI and in project page]
 
-Once found, copy the projectID into the Upstream search bar. If the project exists in your network of connected peers, it will appear. Following the project will replicate it to you local machine.
+Once found, copy the Radicle ID into the Upstream search bar. Following the project will replicate it to your local machine once it's found in your network of connected peers.
 
 [insert picture of projectID & search bar]
 
 ### Cloning projects
 
-To clone a working copy of a followed project to your computer, you can use the [**Checkout**](understanding-radicle/glossary.md/#checkout) option within the Upstream client. This will clone the project to a specified location on your local machine and initialize a Git repository.
+To clone a working copy of a followed project to your computer, you can use the [**Checkout**](understanding-radicle/glossary.md/#checkout) option within the Upstream client. This will clone the project to a specified location on your local machine.
 
-Commiting and publishing changes to this working copy will create a new [remote](understanding-radicle/glossary.md/#remote) for that project that is uniquely linked to your peerID. You can view this remote in the peer switcher.
+Commiting and publishing changes to this working copy will create a new [remote](understanding-radicle/glossary.md/#remote) for that project which is uniquely linked to your peerID. You can view this remote in the peer switcher.
 
 [insert picture of peer switcher]
 
-Once a remote is created for a project, it will also add the project to your **My Projects** tab. 
+Once a remote is created for a project, it will also move the project to your **Projects** tab.
 
-## Getting changes from a project 
+## Getting changes from a project
 
 ### Adding remotes
-To view and pull changes from another user's version of your project, you must first **add their remote**. To do this, you must add their **peerID** to your list of remotes. 
+To view and pull changes from another user's version of your project, you must first **add their remote**. To do this, you must add their **Device ID** to the list of remotes that you follow.
 
-[insert gif of adding peerID to manage remotes modal]
+[insert gif of adding Device ID to manage remotes modal]
 
-Adding a user's remote is basically adding their version of your project to your machine. Once added, you'll be able to fetch changes from their version and apply them to your working copy. Read more about [how remotes work on Radicle](understanding-radicle/faq.md).
+Adding and following a user's remote is basically adding their version of your project to your machine. To get changes from a new remote you must fetch them from a connected peer that has replicated the remote's changes. Once fetched, you'll be able to pull changes from their version and apply them to your working copy. Read more about [how remotes work on Radicle](understanding-radicle/faq.md).
```

Comment:
How's this? The previous version makes it sound like the user can fetch via some button or whatever, but that's not the case.

```suggestion
Adding a peer's remote means that you wish to track that peer's changes for this project. It's important to note that these changes may not instantly appear. We rely on the connectdness of the network to ensure that we hear about the changes from a peer we care about. Assuming that the network of peer's also track the peer you are looking for, you will get their changes when:
1. On startup of the application, where your peer will ask for any new changes from all the peers you track. For more information on tracking [click here](is there a tracking doc?).
2. Your peer receives a gossip message about the peer you wish to track.  For more information on gossip [click here](doc-on-gossip?).
```

Reply:
Great rewrite - made some minor edits to keep language consistent. Can you review @FintanH ?

Adding a peer's remote means that you wish to track their changesets for your project. It's important to note that these changes may not instantly appear. We rely on the connectedness of the network to ensure that we hear about the changes from a peer we follow. Assuming that the network of peers also track the peer you are looking for, you will get their changes:
1. On application startup, where your peer will ask for any new changes from all the peers you follow. For more information on following, [click here](understanding-radicle/how-it-works.md/#following).
2. Whenever your peer receives a gossip message about the peer you wish to follow.  For more information on gossip, [click here](understanding-radicle/how-it-works.md/#gossip).

Reply:
Looks great! :) Maybe one slight modification now that I'm reading it again:

> Assuming that the network of peers also track the peer you are looking for, you will get their changes:

changes to:

> Assuming that the network of peers includes the peer you wish to track _or_ also tracks the peer you are looking for, you will get their changes:

Reply:
Cool, updated your suggestion directly with this edit. @rudolfs - if you want to commit, go ahead and then we can get this merged.

Reply:
https://github.com/radicle-dev/radicle-docs/pull/16/commits/92a6d1db12cefe8ccb135b074e7f5bd4a8d29fd8